src/yourdfpy/urdf.py

```diff
@@ -235,6 +235,18 @@ def _str2float(s):
     return float(s) if s is not None else None
 
 
+def apply_visual_color(geom: trimesh.Trimesh, visual: Visual):
+    if visual.material is None or visual.material.color is None:
+        return
+    n = len(geom.visual.face_colors)
+    int_color = [int(255 * channel) for  channel in visual.material.color.rgba]
+    for i in range(n):
```

Comment:
Just to understand this loop: Wouldn't `geom.visual.face_colors[:] = visual.material.color.rgba` also work?

Reply:
It totally does, and performs better than the lazy Python loop :smiley: